### PR TITLE
GOB: implement 800x600 resolution & set some games to the required resolution

### PR DESCRIPTION
**Pull request by BJNFNE**
commit message should be called: GOB: implement 800x600 resolution & set some games to the required resolution

### CODE REVIEW

File: engines/gob/gob.cpp [hunk 1]
```diff
@@ -699,9 +699,10 @@ void GobEngine::deinitGameParts() {
 }
 
 Common::Error GobEngine::initGraphics() {
-	if        (is800x600()) {
-		warning("GobEngine::initGraphics(): 800x600 games currently unsupported");
-		return Common::kUnsupportedGameidError;
+	if (is800x600()) {
```
Comment (tag2015) [inline engines/gob/gob.cpp:702]: All the games that you marked as 800x600 are currently unsupported, so why this change?

Comment (BJNFNE) [reply engines/gob/gob.cpp:702]: or should i make a new PR to use the correct resolution only in the tables with the 800x600 implementation?

Comment (tag2015) [reply engines/gob/gob.cpp:702]: The purpose of this check is to stop execution for 800x600 games (because they are unsupported) and print a warning. With your change you allow the screen initialization but does any of these games actually work?

Comment (BJNFNE) [reply engines/gob/gob.cpp:702]: i thought because these games are using already 800x600, so i can allow already 800x600 in the Engine for later purposes.

Comment (BJNFNE) [reply engines/gob/gob.cpp:702]: nope Adibou 1 is 320x200 and Adibou 640x480

**Comment by sev-** [comment]
The term "implementation" is not applicable here from any standpoint. Nothing was implemented.

A more applicable term here is "Marked". E.g. you marked these game versions as 800x600.

However, I do not like the newly added and unused flag. I would say that the marking could better go into comments, not as the feature. Then, if somebody has time to work on adding support for these games, they could see these markings in the comments. But before this, I would not just throw a flag that is only referenced but affects nothing.

**Comment by BJNFNE** [comment]
so i decided then to close this PR, and make a new one to add the 800x600 resolution for DEV7 games as a comment
